app.py

```diff
@@ -0,0 +1,67 @@
+import os, sys, json
+
+import boto3, requests, sqlalchemy
+from requests_http_signature import HTTPSignatureAuth
+from chalice import Chalice, Response
+from dcplib.etl import DSSExtractor
+
+from dcpquery import api, config, aws
+from dcpquery.db import run_query
+from dcpquery.exceptions import DCPQueryError
+
+swagger_spec_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), f'{os.environ["APP_NAME"]}-api.yml')
+app = api.DCPQueryServer(app_name=os.environ["APP_NAME"], swagger_spec_path=swagger_spec_path)
+config.app = app
+
+
+@app.route("/")
+def root():
+    body = f'<html><body>Hello from {os.environ["APP_NAME"]}! Swagger UI is <a href="v1/ui">here</a>.</body></html>'
+    return Response(status_code=requests.codes.ok, headers={"Content-Type": "text/html"}, body=body)
+
+
+@app.route("/internal/health")
+def health_check():
+    return json.dumps({"app": str(app), "db": str(config.db)})
+
+
+@app.route("/bundles/event", methods=["POST"])
+def handle_bundle_event():
+    print("Received bundle event:")
+    print(app.current_request.json_body)
+    HTTPSignatureAuth.verify(requests.Request(url="http://host/bundles/event",
```

Comment:
nit: insert comment that the url host component is not used.

Reply:
Done, thanks.